README.md

```diff
@@ -55,14 +55,6 @@ If you haven't done yet, go to [Auth0](https://auth0.com) and create an Account,
 https://{YOUR_AUTH0_DOMAIN}/android/{YOUR_APP_PACKAGE_NAME}/callback
 ```
 
-If you plan to use Passwordless authentication with SMS or Email connections, the Callback URL you will register looks like this:
-
-```
-https://{YOUR_AUTH0_DOMAIN}/android/{YOUR_APP_PACKAGE_NAME}/sms
```

Comment:
when you start a passwordless flow and ask for "android_link" this is the format of callback URL that the server will use. This is not changeable by the developer. 

The server takes the domain from the Auth0 application and the Android package name from this setting:
![image](https://user-images.githubusercontent.com/3900123/116680843-7659d580-a9ac-11eb-80d3-d55939b033fd.png)